config/environment.js

```diff
@@ -101,6 +101,7 @@ module.exports = function (environment) {
       status: 'https://www.traviscistatus.com/',
       support: 'mailto:support@travis-ci.com',
       terms: 'https://docs.travis-ci.com/legal/terms-of-service/',
+      travisvsjenkins: 'https://travis-ci.com/travisci-vs-jenkins',
```

Comment:
```suggestion
      travisVsJenkins: 'https://travis-ci.com/travisci-vs-jenkins',
```
I would suggest camel casing the key here. What do you think?